### ISSUE TITLE
[dev-2.0] AttributeError: module 'jsonschema._validators' has no attribute 'properties_draft4'

### ISSUE THREAD

**Issue by rykerrumsey** [zalando/connexion#707]
### Description
Installed --pre connexion into a pipenv environment
Ran the command openapi-spec-validator

### Expected behaviour
```
usage: openapi-spec-validator [-h] [--schema {2.0,3.0.0}] filename
openapi-spec-validator: error: the following arguments are required: filename
```

### Actual behaviour
```
Traceback (most recent call last):
  File "/home/rykerrumsey/.local/share/virtualenvs/api-8vZEM58Z/bin/openapi-spec-validator", line 7, in <module>
    from openapi_spec_validator.__main__ import main
  File "/home/rykerrumsey/.local/share/virtualenvs/api-8vZEM58Z/lib/python3.6/site-packages/openapi_spec_validator/__init__.py", line 7, in <module>
    from openapi_spec_validator.factories import JSONSpecValidatorFactory
  File "/home/rykerrumsey/.local/share/virtualenvs/api-8vZEM58Z/lib/python3.6/site-packages/openapi_spec_validator/factories.py", line 5, in <module>
    from openapi_spec_validator.generators import (
  File "/home/rykerrumsey/.local/share/virtualenvs/api-8vZEM58Z/lib/python3.6/site-packages/openapi_spec_validator/generators.py", line 12, in <module>
    class SpecValidatorsGeneratorFactory:
  File "/home/rykerrumsey/.local/share/virtualenvs/api-8vZEM58Z/lib/python3.6/site-packages/openapi_spec_validator/generators.py", line 19, in SpecValidatorsGeneratorFactory
    'properties': _validators.properties_draft4,
AttributeError: module 'jsonschema._validators' has no attribute 'properties_draft4'
```

### Steps to reproduce
1. pipenv shell
2. pipenv install --pre connexion
3. openapi-spec-validator

### Additional info:
Output of the commands:

- `python --version`
```Python 3.6.5 :: Anaconda, Inc.```

- `pipenv graph`
```
connexion==2.0.0rc1
  - clickclick [required: >=1.2, installed: 1.2.2]
    - click [required: >=4.0, installed: 7.0]
    - PyYAML [required: >=3.11, installed: 4.2b4]
  - flask [required: >=0.10.1, installed: 1.0.2]
    - click [required: >=5.1, installed: 7.0]
    - itsdangerous [required: >=0.24, installed: 0.24]
    - Jinja2 [required: >=2.10, installed: 2.10]
      - MarkupSafe [required: >=0.23, installed: 1.0]
    - Werkzeug [required: >=0.14, installed: 0.14.1]
  - inflection [required: >=0.3.1, installed: 0.3.1]
  - jsonschema [required: >=2.5.1, installed: 3.0.0a3]
    - attrs [required: >=17.4.0, installed: 18.2.0]
    - pyrsistent [required: >=0.14.0, installed: 0.14.4]
      - six [required: Any, installed: 1.11.0]
    - six [required: >=1.11.0, installed: 1.11.0]
  - openapi-spec-validator [required: >=0.2.4, installed: 0.2.4]
    - jsonschema [required: Any, installed: 3.0.0a3]
      - attrs [required: >=17.4.0, installed: 18.2.0]
      - pyrsistent [required: >=0.14.0, installed: 0.14.4]
        - six [required: Any, installed: 1.11.0]
      - six [required: >=1.11.0, installed: 1.11.0]
    - PyYAML [required: Any, installed: 4.2b4]
    - six [required: Any, installed: 1.11.0]
  - PyYAML [required: >=3.11, installed: 4.2b4]
  - requests [required: >=2.9.1, installed: 2.19.1]
    - certifi [required: >=2017.4.17, installed: 2018.8.24]
    - chardet [required: >=3.0.2,<3.1.0, installed: 3.0.4]
    - idna [required: >=2.5,<2.8, installed: 2.7]
    - urllib3 [required: >=1.21.1,<1.24, installed: 1.23]
  - six [required: >=1.9, installed: 1.11.0]
```

**Comment by dtkav** [comment]
Thanks for the ticket. I'm guessing the problem is with jsonschema 3.0.0a3.
I'll have a look. We probably just need to be stricter on the version dependency.

**Comment by dtkav** [comment]
Thanks @rykerrumsey ! Please have a look at #708 when you get a chance.

**Comment by dtkav** [comment]
Thanks! I've merged in #708 , so it should be part of the next RC.

**Comment by mattixpet** [comment]
It happened to me getting this error, and tldr; upgrading Connexion version to 2.14.0 solved it. As the solution linked here solves this issue. 

However, I was using swagger-ui and thought it was some different version of Connexion, was using 2.7.0 (thought it was 2.70.0 (higher than 2.14), or some other number since it used swagger, installed like `connexion[swagger-ui]==2.7.0`). Anyway, I was being dumb and changing to `connexion[swagger-ui]==2.14.0` solved this.
